src/App.tsx

```diff
@@ -1,156 +1,103 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
 
-const USER_ID = 0;
+import { getTodos } from './api/todos';
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+import { Errors } from './types/Errors';
+import { Filters } from './types/Filters';
+import { TodoType } from './types/TodoType';
+
+import TodoList from './Components/TodoList/TodoList';
+import Notification from './Components/Notification/Notification';
+import Footer from './Components/Footer/Footer';
+import Header from './Components/Header/Header';
+
+const USER_ID = 6535;
+
+function filteredTodos(filter: Filters, todos: TodoType[]) {
+  switch (filter) {
+    case Filters.All:
+      return todos;
+
+    case Filters.ACTIVE:
+      return todos.filter((todo: TodoType) => !todo.completed);
+
+    case Filters.COMPLETED:
+      return todos.filter((todo: TodoType) => todo.completed);
+
+    default:
+      return todos;
   }
+}
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<TodoType[]>([]);
+  const [isError, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState<Errors | null>(null);
```

Comment:
You don't need `isError` state. Just check if errorMessage is empty, if true - you don't have one, if not - you have an error.